bot/src/main/java/com/kantenkugel/discordbot/versioncheck/items/VersionedItem.java

```diff
@@ -24,6 +24,13 @@
      */
     public abstract String getName();
 
+    /**
+     * The description of this version item, used for listing purposes.
+     *
+     * @return  The description
+     */
+    public abstract String getDescription();
```

Comment:
could throw some exception by default so that it isn't nullable and the `return null` implementations ones aren't needed